README.md

```diff
@@ -0,0 +1,101 @@
+<h2 align="center">Inside Server</h2>
+
+<p align="center">
+  <a href="https://github.com/ifactory-solutions/inside-server/stargazers">
+    <img alt="GitHub Stars" src="https://img.shields.io/github/stars/ifactory-solutions/inside-server.svg?style=social&label=Star">
+  </a>
+  <a href="https://github.com/ifactory-solutions/inside-server/issues">
+    <img alt="GitHub Issues" src="https://img.shields.io/github/issues/ifactory-solutions/inside-server.svg?style=flat-square">
+  </a>
+  <a href="https://github.comifactory-solutions/inside-server/pulls">
+    <img alt="GitHub Pull Requests" src="https://img.shields.io/github/issues-pr-raw/ifactory-solutions/inside-server.svg?style=flat-square">
+  </a>
+  <a href="https://www.codacy.com/app/jourdan.rodrigues/inside-server?bid=5482373&utm_source=github.com&amp;utm_medium=referral&amp;utm_content=ifactory-solutions/inside-server&amp;utm_campaign=Badge_Grade">
+    <img alt="Codacy" src="https://api.codacy.com/project/badge/Grade/5d3d3292927b4b558d418999b135f5e6?branch=master">
+  </a>
+  <a href="https://travis-ci.org/ifactory-solutions/inside-server">
+    <img alt="Build Status" src="https://img.shields.io/travis/ifactory-solutions/inside-server.svg?style=flat-square">
+  </a>
+  <a href="http://makeapullrequest.com">
+    <img alt="PRs Welcome" src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square">
+  </a>
+</p>
+
+Inside is an open source platform to deal with all the intern process of a company related to its employees. Our goal is to make easy to keep a track and manage all information related to the people that work **inside** the company.
+
+<p align="center">
+  <a href="" target="_blank">
```

Comment:
As you can see, the text after `target="_blank"` seems like "broken". So you should put a `\` before `_`, because `_` is for _emphasis_ in Markdown syntax.

Reply:
Pushing a fix right away.